src/community/ogcapi/dggs/dggs-core/src/main/java/org/geotools/dggs/gstore/DGGSGeometryStoreFactory.java

```diff
@@ -46,14 +46,14 @@
     @Override
     public DataStore createDataStore(Map<String, ?> params) throws IOException {
         String factoryId = (String) DGGS_FACTORY_ID.lookUp(params);
-        DGGSInstance instance = DGGSFactoryFinder.createInstance(factoryId, params);
-        DGGSGeometryStore store = new DGGSGeometryStore(instance);
-
-        String namespace = (String) NAMESPACE.lookUp(params);
-        if (namespace != null) {
-            store.setNamespaceURI(namespace);
+        DGGSGeometryStore store;
+        try (DGGSInstance instance = DGGSFactoryFinder.createInstance(factoryId, params)) {
```

Comment:
Nope, the instance is wrapped by the store and managed there, should not be closed.

Reply:
Removed from the PR.